cmd/secretmessage/main.go

```diff
@@ -61,7 +61,11 @@ func initConfig() *BotConfig {
 	}
 
 	config.Database.URL = fmt.Sprintf(
-		"postgres://%s:%s@%s/%s", config.Database.Username, config.Database.Password, config.Database.Host, config.Database.Name,
+		"postgres://%s:%s@%s/%s",
+		os.Getenv(config.Database.Username),
```

Comment:
`os.Getenv()` is waiting for the env var key but here you pass a value which may be empty according to the change from config to env var

Reply:
Nope, I pass the env key as specified in the README. So that the key can be change easily without modifying the code.